### ISSUE TITLE
Passing options to `sixpack.Session()` does not work in node v4

### ISSUE THREAD

**Issue by timelf123** [seatgeek/sixpack-js#11]
Stack is Node v4
```
var session = new sixpack.Session({client_id: req.user ? req.user._id : sixpack.generate_client_id, base_url: 'http://XXX:8080'});

```
`console.log(session)`
```
{ client_id: { client_id: [Function], base_url: 'http://XXX:8080' },
  base_url: 'http://localhost:5000',
  ip_address: null,
  user_agent: null }
```

The options object seems to only be injected into client_id. I've tried just setting base_url too. This seems to work fine in the browser, with a quick jsfiddle here: http://jsfiddle.net/qpfac336/

**Comment by timelf123** [comment]
After setting my package.json file to the latest commit, my issue was solved (missing a polyfill implementation of `Object.Assign()` that is not available in 1.0.0). Thanks for fixing it :)

```
"sixpack-client": "git+ssh://git@github.com/seatgeek/sixpack-js.git#74bcfc137a85a34fa19c4d5ef1fcd04525a05218",
```


tl;dr: a bumped package version would make `npm install sixpack-js` installs work by default
